tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc

```diff
@@ -1587,7 +1587,7 @@ bool ral_conv_bladnn(ExecutionContext* ctx, void* stream_handle,
     ki = kernel.sizes[3];
     oh = output.sizes[1];
     ow = output.sizes[2];
-    oc == output.sizes[3];
+    oc = output.sizes[3];
```

Comment:
nit: it's not a bug of stream executor. maybe refine the description of your PR.

Reply:
Done